### ISSUE TITLE
Staging Review finding: Helper text not likely to be read by screen reader

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82852]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Pension benefits
**Product:** 527EZ
**Feature:** Update multi-response (single page list loop) pattern

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/va-treatment/medical-centers](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/medical/history/va-treatment/medical-centers), the text "Enter all VA medical centers where you have received treatment" is not currently announced by screen readers when tabbing to the first input. This isn't super critical --- I think most users will understand from context what they're supposed to do, and they do hear it when focus is set to that text when an item is removed --- but it's a good practice to ensure screen reader users hear these instructions on their first pass.

The `fieldset` containing the inputs has `legend` text of "VA medical centers," and the "Enter all..." text is in a second `legend`. Multiple legends in one `fieldset` aren't typically supported, unless it's a nested-`fieldset` situation.

**Note:** This issue is also present on [/federal-treatment/medical-centers](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/medical/history/federal-treatment/medical-centers) and [/current/history](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/employment/current/history)

<details><summary>Link, screenshot or steps to recreate</summary>

Current markup goes something like:
```
<fieldset ...>
	<legend ...>
		<h3 ...>VA medical centers</h3>
	</legend>
	<div>
		<div ...>
			<div ...>
				<legend ... id="root_vaMedicalCenters__title">Enter all VA medical centers where you have received treatment</legend>
			</div>
			<div class="va-growable">
				[cards]
			</div>
		</div>
	</div>
</fieldset>
```

</details>

## Recommended action

My best recommendation is to combine the two `legend` elements and then style with CSS as appropriate, eg.
```
<fieldset ...>
	<legend ...>
		<h3 ...>VA medical centers</h3>
		<span id="root_vaMedicalCenters__title">Enter all VA medical centers where you have received treatment</span>
	</legend>
	<div>
		<div ...>
			<div class="va-growable">
				[cards]
			</div>
		</div>
	</div>
</fieldset>
```

If that's not possible for some reason, please feel free to ping me in Slack and I can help come up with another option.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
